astro/src/content/docs/apis/identity/verify.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Identity Verify
+description: Learn about the APIs for starting, sending, and completing identity verification.
+section: apis
+---
+import API from 'src/components/api/API.astro';
+import APIBlock from 'src/components/api/APIBlock.astro';
+import APIField from 'src/components/api/APIField.astro';
+import Aside from 'src/components/Aside.astro';
+import InlineField from 'src/components/InlineField.astro';
+import JSON from 'src/components/JSON.astro';
+import StandardPostResponseCodes from 'src/content/docs/apis/_standard-post-response-codes.astro';
+import XFusionauthTenantIdHeaderScopedOperation from 'src/content/docs/apis/_x-fusionauth-tenant-id-header-scoped-operation.mdx';
+
+## Overview
+
+<Aside type="version">
+  This API has been available since 1.99.9
+</Aside>
+
+This page contains the APIs for verifying the identity of a User. This includes starting the verification process, sending verification requests, and completing the verification. This is similar to the [Verify Email](/docs/apis/users#verify-a-users-email) API, but it supports all verifiable identity types (email and phone number).
+
+When using these APIs, the general workflow is to call the start API, optionally call the send API to have FusionAuth send the verification message to the User, then call the complete API based on what the User has provided to complete verification. You will not often need to call these APIs directly but the main use cases for these APIs are:
+
+| Use Case                                              | Start API                | Send API                | Complete API                                                                                                   |
```

Comment:
try to help people understand when they should care about this stuff

Reply:
also assumes some variant of what we talked about with ENG-2714 in refinement this AM.